x-team/xp-ui
# Update zindex for InputField > svg

**Release Type:** *Bug Fix* 


Fixes https://x-team-internal.atlassian.net/browse/XP-3776

## Description

When we select **I'll be available on a certain date** for the **What's your availability like right now?** field on the applicant Info page (/signup/info), any subsequent attempt to select a different option from that dropdown control gives UI glitch.

The problem was an incorrect z-index defined for **InputField > svg**

## Checklist

**Before submitting a Pull Request,** please make sure the following is done:



## Steps to Test or Reproduce

- In **XP-REGISTRATION**:
1. Download first this **XP-UI** change and make sure you are using it for your local XP-REGISTRATION project
2. Go to the http://localhost:8001/signup/info page 
3. Select **I'll be available on a certain date** for the **What's your availability like right now?** field
4. Now try to select another option for the **What's your availability like right now?** field and confirm that you can do it in a cleaned way without having the date icon over the select options

## Impacted Areas in Application

- src/components/forms/InputField.js

## Screenshots

![Screen Capture on 2020-06-03 at 13-19-31](https://user-images.githubusercontent.com/6010411/83668551-273dd180-a59e-11ea-95a5-f8f28317f34e.gif)
